### DISCUSSION TITLE
High Level Overview of all the main Types in Wikind?

### DISCUSSION THREAD

**Discussion by lancejpollard** [uwu-tech/Kind#424] [Q&A] [answered]
I am looking at [Wikind](https://github.com/kindelia/wikind) and the let's say ~40 folders, each which have nested folders and files, and wondering if you could explain the reasoning for including such types in the foundation. I'm curious how it all fits together and why these pieces specifically are all needed instead of other things or ways of organizing stuff. Would be cool to get more introduction to the overall model.

**Comment by algebraic-dev** [comment, marked as answer]
All files in Wikind are affected by the fact that capitalized identifiers in Kind2 such as 'I.Like.Pudding' are converted to paths, e.g. 'I/Like/Pudding.kind2' if Kind2 cannot find the entry in the current "book". We use this approach because we do not need to build everything, just the files we use rn. Currently we are writing a guide for the whole language, but Kind is still far from a complete language, as we are trying to move fast and fix problems that breaks the entire Wikind.
